### DISCUSSION TITLE
Need to change redirect paths of login after successful redirection and on back to application button after completing forgot password

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Ed/Community#742] [Sunbird ED configuration discussion]
Hi team,
Hope you are doing well!
**We have a requirement from client where we want to redirect users after every login session to another page than "/resources"**
We have completed scenarios of login button, login button on registration page etc
We are unable to change redirect path for 2 scenarios:
1. Login redirect after successful registration
2. After completing forgot password flow, We get "Back To Application" button. Need to change href path of it

**- Is it possible to change this ? If yes, How can we update achieve this.?Any configurations should be updated?**

We appreciate any insights. Thank you!

@vinukumar-vs ,@rajeevsathish, @AmiableAnil , @Krishnaj20, @chitranshu-keshav, @SanthoshVasabhaktula, @gandham-santhosh , @coolbung @komalm @manishakapadnis, @vky25, @Vinaysv18,

**Comment by pranavirabatti** [comment]
@paritshivani I’m not 100% sure about this, but you can configure the redirect URIs in Keycloak under the 'Clients' section. Additionally, in the themes, you can modify the URL for the forgot password flow. Please correct me if I’m wrong, anyone, and check with someone who is experienced with the Keycloak tool for further help.

**Comment by HarishGangula** [comment]
@paritshivani 
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
